### PR TITLE
Adds moths

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: Adds Moths! They're a new species that have (unusable due to gravity and shit) wings, and are prone to screaming. Alot.
code: Added a way for screaming sounds to be added to species
/:cl:

Why: 
  

**CREDITS TO HIPPIE'S @tortellinitony**

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -394,6 +394,7 @@ ROUNDSTART_RACES human
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
 ROUNDSTART_RACES plasmaman
+ROUNDSTART_RACES moth
```
Comment (Cyberboss) [inline config/game_options.txt]: OFF

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: config/game_options.txt (hunk 1 above)
Comment (MoreRobustThanYou) [reply config/game_options.txt]: why? they're no better than humans, they just look stupid

**Comment by epochayur** [comment]
make their sole food source fruit, sugar water and clothing and I'm sold

**Comment by praisenarsie** [comment]
# credits to hippie i guess

**Comment by Spaceinaba** [comment]
HOT

**Review by praisenarsie — CHANGES_REQUESTED** [review]
make a better changelog

**Comment by MoreRobustThanYou** [comment]
pictures:

File: config/game_options.txt (hunk 1 above)
Comment (Cyberboss) [reply config/game_options.txt]: # O F F

Comment (praisenarsie) [reply config/game_options.txt]: do what the maintainer says man.

Comment (KorPhaeron) [reply config/game_options.txt]: off

Comment (KorPhaeron) [reply config/game_options.txt]: Dont need more cosmetic roundstart races, these can be silly things from xenobio or jungle dweller ghost role or something

**Comment by Spaceinaba** [comment]
moths should have to sustain themselves by eating clothing like in real life and also run into lightbulbs

File: config/game_options.txt [hunk 2]
```diff
@@ -390,11 +390,11 @@ SILICON_MAX_LAW_AMOUNT 12
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
 ROUNDSTART_RACES human
 
-## Races that are strictly worse than humans that could probably be turned on without balance concerns
+## Races that are strictly worse than humans that could probably be turned on without balance concerns, but will probably be off due to the strict NO FUN policy
```
Comment (praisenarsie) [inline config/game_options.txt]: remove this line, also we have a fucking config on the server. we don't use this config.

Comment (Cyberboss) [reply config/game_options.txt]: Do you want me to close this?

Because this is how you get me to close this.

Comment (MoreRobustThanYou) [reply config/game_options.txt]: it's true though

**Comment by lzimann** [comment]
no thanks

**Comment by MoreRobustThanYou** [comment]
@lzimann how dare you close something that kor (and sherrie) requested?

**Comment by KorPhaeron** [comment]
It's harmless and will make players happy

It's the shallow kind of content that people love to see, something shiny and new without balance implications

**Comment by Cyberboss** [comment]
Remove the passive aggression or i'm not reopening it

**Comment by Kraseo** [comment]
imagen

**Comment by GuyonBroadway** [comment]
Moths are universally cuckery shitlers and here is the proof.

https://youtu.be/126Jb3mdGwE

File: config/game_options.txt (hunk 2 above)
Comment (KorPhaeron) [reply config/game_options.txt]: this is why you keep getting permabanned everywhere morty

**Comment by Armhulen** [comment]
What's with his eyes

**Comment by MoreRobustThanYou** [comment]
Passive-aggressive comment removed

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -244,6 +244,15 @@
...
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/scream/run_emote(mob/user, params)
+	. = ..()
+	if(!.)
+		return ..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.dna.species.scream_sound)
+			playsound(H.loc, H.dna.species.scream_sound, 50, 1, 4, 1.2)
```
Comment (optimumtact) [inline code/modules/mob/living/emote.dm]: this is a terrible idea

**Comment by FuryMcFlurry** [comment]
YOU CREDITED THE WRONG PERSON

**Comment by TDcoolguy300** [comment]
WRRRRRRRY HES RIGHT
FURY MADE THE SPRITES
I DID THE ORIGINAL IMPLEMENTATION
TORTILINI PORTED IT TO MODERN HIPPIE
REEEEEEEEEEEEE

**Pull request closed without merging**